source/style-guide/markup/inline.txt

```diff
@@ -0,0 +1,119 @@
+=============
+Inline Markup
+=============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Inline markup, also known as roles, specify how the enclosed text should be
+interpreted or rendered.
+
+Learn more about roles from the `{+docutils+} documentation <http://www.docutils.org/docs/ref/rst/roles.html>`__.
+
+{+sphinx+} provides an extended set of roles. Learn more about these roles
+from the `Sphinx documentation <https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html>`__.
+
+Font Formatting Roles
+---------------------
+
+These roles control how the font of the enclosed text is rendered.
+
+Learn more about when to use these roles from the style guide section on
+:ref:`<text-formatting>`.
+
+- To **strongly emphasize** or **bold** text, use two asterisks on
```

Comment:
[question]

Is there anywhere in the style guide that provides guidance on when to use bold and italics formatting? I understand the purpose of this section to focus on how to format text, but maybe in future work we should add a bit more context as to when and why to use each.

Reply:
Yes, [it's referenced in lines 24-25](https://github.com/mongodb/docs-meta/pull/128/files#diff-4775599d9eec9251fcfced3eecfd751a556fbc4aa4e814b345ce84429afc2d07R24-R25) above which is an addition I made to the content of the old PR. I think the "markup/" path was intended only to show how to write the rST.

Reply:
Got it, thanks.